src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,54 +1,128 @@
-import React from 'react';
+import React, { Dispatch, SetStateAction, useState } from 'react';
 import './FindMovie.scss';
+import classNames from 'classnames';
+import { getFilmByTitle } from '../../API/api';
+import { MovieCard } from '../MovieCard';
 
-// import { MovieCard } from '../MovieCard';
-
-export const FindMovie: React.FC = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+type Props = {
+  selectedMovie: Dispatch<SetStateAction<Movie[]>>,
+  movies: Movie[],
+};
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie: React.FC<Props> = ({ selectedMovie, movies }) => {
+  const [title, setTitle] = useState('');
+  const [preview, setPreview] = useState<Movie | null>(null);
+  const [error, setError] = useState(false);
+  const [duplicate, setDuplicate] = useState(false);
+
+  const findMovie = () => {
```

Comment:
I don't shure, but didn't JS prevent to name variables/functions with already used names in the same component? You have used `findMovie `  2 times.

Reply:
the syntax is differend

Reply:
oh I see! yeh, first later is capital in component!